homeassistant/components/myio/config_flow.py

```diff
@@ -0,0 +1,146 @@
+"""Config flow for myIO integration."""
+import logging
+
+from myio.comms_thread import CommsThread  # pylint: disable=import-error
+from slugify import slugify
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+
+from .const import CONF_PORT_APP, DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+COMMS_THREAD = CommsThread()
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, default="myIO-Server"): str,
+        vol.Required(CONF_HOST, default="192.168.1.170"): str,
```

Comment:
```suggestion
        vol.Required(CONF_HOST): str,
```

This default will only work for you.

Reply:
this IP address is the factory default of myIO server.

Reply:
Still, my network is `192.168.2.x` so this will not work for me, this is specific for each network setup and should not be a default.

Reply:
this is a default, most of the peoples use this... You can change it whatewer you want at the config flow...